docs/manifest.json

```diff
@@ -1019,6 +1019,18 @@
 		"markdown_source": "../packages/components/src/isolated-event-container/README.md",
 		"parent": "components"
 	},
+	{
+		"title": "ItemGroup",
+		"slug": "item-group",
+		"markdown_source": "../packages/components/src/item-group/item-group/README.md",
+		"parent": "components"
+	},
+	{
+		"title": "Item",
+		"slug": "item",
+		"markdown_source": "../packages/components/src/item-group/item/README.md",
+		"parent": "components"
```

Comment:
I wonder if this should be `"parent": "item-group"`. But I'm not sure if this level of nesting actually works or is desirable in the Handbook. @gziolo Any thoughts?

Reply:
The structure in the handbook is rather flat for components. So I would leave it as is.

The other question is whether `<Item />` really depends on the `<ItemGroup />`. In other cases grouping is optional.

Reply:
`Item` is quite dependant on `ItemGroup` at the moment — for example, it reads the `ItemGroup` context in order to derive values for the `size` prop.

On its own, it's not very feature-full, nor opinionated in terms of styling. It really just is a `div[role="listitem"]` which renders a `div` or a `button` depending on the `isAction` prop.